test/README.md

```diff
@@ -87,6 +87,43 @@ environment.
    docker is used only for running drenv tests locally. You can use
    podman for building and running containers locally.
 
+1. Install `podman`
+
+   ```
+   sudo dnf install podman
+   ```
+
+   See [Installing podman](https://podman.io/docs/installation) for other options.
+
+## Troubleshooting drenv issues
+
+### podman commands are failing for rootless access
+
+```
+$ podman version
+ERRO[0000] running /usr/bin/newuidmap 61879 0 1000 1 1 100000 65536: newuidmap:
+open of uid_map failed: Permission denied
+Error: cannot set up namespace using "/usr/bin/newuidmap": exit status 1
+```
+
+While running with root access it works fine.
+
+```
+$ sudo podman version
+Client:       Podman Engine
+Version:      4.6.2
+API Version:  4.6.2
+Reported with Podman version 4.6.2 on Fedora 37.
+```
```

Comment:
The code example of running podamn version is not needed.

The text "Reported with Podman version 4.6.2 on Fedora 37." can be moved to the description of the problem.